### PR TITLE
Feature/calendar

### PR DESCRIPTION
**Pull request by ben-me**
- adjusted a lot of styling. Added primary color and background color as variables in global styles. Moved a lot of stuff around
- installed react-datepicker
- created calendar component and modified it (no days in the past, no weekends, portal opened)
- implemented and rendered the calendar
- created useState to keep track of the selected day
- added a picture for the homepage

### CODE REVIEW

File: pages/index.js [hunk 1]
```diff
@@ -33,8 +47,9 @@ const Anchor = styled.a`
 `;
 
 const Container = styled.div`
-  display: flex;
-  align-items: center;
-  justify-content: center;
+  /* display: flex; */
+  position: relative;
+  /* align-items: center;
+  justify-content: center; */
```
Comment (K-Juno) [inline pages/index.js]: You should remove comments.

**Comment by K-Juno** [comment]
I see that you managed to exclude weekends, which is a cool option! Did you by any chance come across a way to exclude public holidays as well (of course that depends on the country then)? Other than that, I'm looking forward to further functionalities when it comes to the Rooms! :)

File: pages/index.js (hunk 1 above)
Comment (ben-me) [reply pages/index.js]: Done. Thanks :)

**Comment by ben-me** [comment]
> I see that you managed to exclude weekends, which is a cool option! Did you by any chance come across a way to exclude public holidays as well (of course that depends on the country then)? Other than that, I'm looking forward to further functionalities when it comes to the Rooms! :)

sadly i haven't found an easy solution for the holidays yet. Maybe I'll do this at a later step but right now it is more important to bring the app forward. I have thought about it though  😁

**Pull request merged**